homeassistant/components/unifi/sensor.py

```diff
@@ -192,6 +199,96 @@ def async_device_state_value_fn(hub: UnifiHub, device: Device) -> str:
     return DEVICE_STATES[device.state]
 
 
+@callback
+def async_device_wan_latency_supported_fn(
+    wan: Literal["WAN", "WAN2"],
+    monitor_target: str,
+    hub: UnifiHub,
+    obj_id: str,
+) -> bool:
+    """Determine if an device have a latency monitor."""
+
+    if (device := hub.api.devices[obj_id]) and device.uptime_stats is None:
+        return False
+
+    return async_device_wan_latency_target(wan, monitor_target, device) is not None
+
+
+@callback
+def async_device_wan_latency_value_fn(
+    wan: Literal["WAN", "WAN2"],
+    monitor_target: str,
+    hub: UnifiHub,
+    device: Device,
+) -> int | None:
+    """Retrieve the monitor target from WAN monitors."""
+
+    target = async_device_wan_latency_target(
+        wan=wan, monitor_target=monitor_target, device=device
+    )
+
+    if target is None:
+        return None
```

Comment:
@Kane610 I donøt think this will happen ever as `async_device_wan_latency_supported_fn` will return `False`

Reply:
I guess its because supported catches it?

Reply:
So maybe supported does it better than I thought, so value_fn should not be expected to return None at all